server/src/main/java/io/druid/server/http/HistoricalResource.java

```diff
@@ -52,4 +52,15 @@ public Response getLoadStatus()
   {
     return Response.ok(ImmutableMap.of("cacheInitialized", coordinator.isStarted())).build();
   }
+
+  @GET
+  @Path("/readiness")
+  public Response getLoadStatusCode()
```

Comment:
Maybe we should change this to `getReadiness` for conformity?

Reply:
Sure, my bad of not changing it myself